tests/unix/cdtest.ml

```diff
@@ -16,7 +16,7 @@
 
 open Lwt.Infix
 
-let port = 9192
+let port = 16_384 + Random.int 10_000
```

Comment:
Was about to say this looks good, but then this always returns 25728. I think you need to add this before it: `let () = Random.self_init ()`